dnsleaktest_FreeBSDjail.sh

```diff
@@ -41,7 +45,10 @@ program_exit ping
 program_exit jq
 check_internet_connection
 
-id=$(shuf -i 1000000-9999999 -n 1)
+#id=$(shuf -i 1000000-9999999 -n 1)
```

Comment:
I think you should use the following code. what you think?

if hash shuf 2>/dev/null; then
    id=$(shuf -i 1000000-9999999 -n 1)
else
    id=$(jot -w %i -r 1 1000000 9999999)
fi

Reply:
This works well, withouth having to rename the dnsleaktest.sh ;)